configure

```diff
@@ -15129,9 +15129,9 @@ LDVERSION='$(VERSION)$(ABIFLAGS)'
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $LDVERSION" >&5
 $as_echo "$LDVERSION" >&6; }
 
-# On Android the shared libraries must be linked with libpython.
+# On Android and Cygwin the shared libraries must be linked with libpython.
 
-if test -z "$ANDROID_API_LEVEL"; then
+if test -z "$ANDROID_API_LEVEL" -o "$MACHDEP" != "cygwin"; then
```

Comment:
This should be `-a` not `-o` technically, but it would probably be even clearer if the test weren't negated, and were instead

```
if test -n "$ANDROID_API_LEVEL" -o "$MACHDEP" = "cygwin";
```

Reply:
"even clearer if the test weren't negated" yeah, please invert the test conditions to avoid double negation ;-)

Reply:
Glad we're on the same page.  It's the double-negation that tripped me up in the first place, and probably you too.

Reply:
> probably you too.

yes